File: Java/GeometricObject.java
package geoms;

public class GeometricObject {
	
	private String color;
	private boolean filled;
	

	public GeometricObject(String color, boolean filled){
		this.color = color;
		this.filled = filled;
	}
	
	public String getColor() {
		return this.color;
	}
	public void setColor(String color) {
		this.color = color;
	}
	public boolean isFilled() {
		return this.filled;
	}
	public void setFilled(boolean filled) {
		this.filled = filled;
	}

	@Override
	public String toString() {
		return "GeometricObject [color=" + color + ", filled=" + filled + "]";
	}
	
}

File: C++/GeometricObject.cpp
#include <iostream>
#include <string>

class GeometricObject{
    private:
        std::string color;
        bool filled;
    public:
        GeometricObject(std::string color, bool filled);
        std::string getColor();
        void setColor(std::string color);
        bool isFilled();
        void setFilled(bool filled);
        std::string toString();
};
GeometricObject::GeometricObject(std::string color, bool filled){
    this ->color = color;
    this -> filled = filled;
}
std::string GeometricObject:: getColor(){
    return this->color;
}
void GeometricObject::setColor(std::string color){
    this->color = color;
}
bool GeometricObject::isFilled(){
    return this->filled;
}
void GeometricObject::setFilled(bool filled){
    this->filled = filled;
}
std::string GeometricObject::toString(){
    const char * filler = (this->filled)? "True": "False";
    return "Geometric Object[ color: " + this->color + ", filled: " + filler +"]";
}
int main(){
    GeometricObject geo("blue", true);
    std::cout << geo.toString();
}